en/user_guide/user.rst

```diff
@@ -0,0 +1,42 @@
+User, Group and Role
+====================
+
+Open Orchestra offers the possibility to manage different users with different rights.
+
+User Management
+---------------
+
+Page users in administration allow to search, create, edit and delete site users.
+
+At the creation, some fields are required :
+
+* first Name
+* last Name
+* e-mail
+* username
+* password
+
+Once the user is created, you can add some information regarding his default language and his group.
+
+Group Management
+----------------
+
+In Open Orchestra, users can be members of groups.
+A group gives a user some rights and access for website administration.
```

Comment:
Actually rights can also be given to access some private parts of the front sites, not only on Back Office.